test/test_reads_at_locus.py

```diff
@@ -180,3 +185,32 @@ def test_reads_at_locus_substitution_shorter():
         base0_read_position_before_variant=0,
         base0_read_position_after_variant=2)
     assert_equal_fields(read, expected)
+
+def test_reads_at_locus_dataframe():
+    sam_all_variants = load_bam("data/b16.f10/b16.combined.bam")
+
+    n_reads_expected = 0
+
+    sam_path_single_variant = data_path(
+        "data/b16.f10/b16.f10.127a.aldh1b1.chr4.45802539.refG.altC.sam")
+    with open(sam_path_single_variant) as f:
+        for line in f:
+            if line.startswith("HWI"):
+                n_reads_expected += 1
+    # we know from inspecting the file that *one* of the reads overlapping this
+    # variant has a CIGAR string of N at the location before and thus we'll
```

Comment:
what do you mean "at the location before", and why (excuse my lack of knowledge here) would it then be missing (presumably, you mean missing from the object returned by reads_at_locus_dataframe below?)?